vllm/worker/worker_base.py

```diff
@@ -23,6 +23,7 @@ class WorkerBase(ABC):
     different hardware. Also abstracts control plane communication, e.g., to
     communicate request metadata to other workers.
     """
+    use_spmd_worker: bool
```

Comment:
can we do the dispatch at the executor level, or at the ray worker wrapper level? I don't think a worker needs to know if it is spmd.

Reply:
Done.